open-craft/edx-platform
# SE-2866 Merge pull request #180 from edx/sustaining-xsslint-security-fixes

Cherry picked from commit edd9b02d520f7bf4dadc110ab46455556325dc93

**Testing instructions**

Using the [test instance](https://manage.opencraft.com/instance/20000/edx-appserver/13111/), run OpenCraft's [manual instance checklist](https://gitlab.com/opencraft/documentation/private/blob/jill/clarify-checklist/checklists/manual_instance_test.md#how-to-elevate-a-user) to ensure basic functionality is unaffected by this change.

**Author Notes & Concerns**

1. Since this change was already merged upstream, we don't have to check each individual change here.

**Reviewer**
